src/pathfinding_service/api.py

```diff
@@ -353,6 +353,7 @@ def post(
 
 class InfoResource(PathfinderResource):
     version = pkg_resources.get_distribution("raiden-services").version
+    contracts_version = pkg_resources.get_distribution("raiden-contracts").version
```

Comment:
There's also `get_contracts_deployment_info` which returns a `DeployedContracts` instance which contains a `contracts_version` attribute: https://github.com/raiden-network/raiden-contracts/blob/master/raiden_contracts/contract_manager.py#L43

Not sure if this is better though, but at least atm we can have contract 0.33 but use the 024 contracts.

Reply:
The python package version is the more visible part (it's in the dependencies), so I definitively want to have that info. I could add the other number, too, but I'm not sure that extra information outweights the potential confusion of showing two version numbers.

Maybe I'll open up the topic of contracts versioning again and see if we can find an easier to understand mapping between those two version numbers, so that giving one of them is usually sufficient.

Reply:
Makes sense.